### ISSUE TITLE
directory content attribute not defined

### ISSUE THREAD

**Issue by annevk** [InternetExplorer/directory-upload#17]
I'm assuming from the examples you want `<input type=file directory>` to do something.

**Comment by aliams** [comment]
It should work in the same way that the `multiple` attribute works. As long as it's set, it will be active.

**Comment by domenic** [comment]
Someone should probably spec that then.

**Comment by aliams** [comment]
It is mentioned in [Section 2.2.1](https://wicg.github.io/directory-upload/proposal.html#widl-HTMLInputElement-directory) under the directory attribute definition. Do you feel that it requires further clarification?

**Comment by domenic** [comment]
That specifies the directory IDL attribute, not the directory content attribute.

**Comment by domenic** [comment]
I'd suggest looking at how attributes (of both kinds) are defined for inputs in the actual HTML spec, and phrasing your spec as a patch to it (see also #15). E.g.:
- [Accept content attribute](https://html.spec.whatwg.org/#attr-input-accept)
- [Accept IDL attribute](https://html.spec.whatwg.org/#dom-input-accept)

**Comment by domenic** [comment]
You also can't do things like "If this attribute is set, input.files must return null." You have to write a proper patch for the definition of [the files IDL attribute](https://html.spec.whatwg.org/#dom-input-files).

**Comment by arunranga** [comment]
@annevk: Do you think the HTMLInputElement should have a Directory Upload state alongside a File Upload state? It makes managing spec'd _activation behavior_ easier than explaining everything within the File Upload activation steps. If you approve, that's what'll go into my PR for HTML.

**Comment by annevk** [comment]
I don't think we can do that since the state is closely tied to the type. We'll need language similar to what we have for the multiple attribute I think.

And some language about the control representing selected files and directories, or some such.
